Zend/zend_inheritance.c

```diff
@@ -810,6 +810,15 @@ static zend_always_inline inheritance_status do_inheritance_check_on_method_ex(z
 	uint32_t parent_flags = parent->common.fn_flags;
 	zend_function *proto;
 
+	if (UNEXPECTED((parent_flags & ZEND_ACC_PRIVATE) && !(parent_flags & ZEND_ACC_ABSTRACT) && !(parent_flags & ZEND_ACC_CTOR))) {
+		if (!check_only) {
+			child->common.fn_flags |= ZEND_ACC_CHANGED;
+			child->common.prototype = NULL;
```

Comment:
Does this line do anything, functionally?

Reply:
I think it came from my original implementation of this RFC, but it's strange that I had kept it. Either way, it seems unnecessary so I removed it.